src/pages/academy/grading/subcomponents/GradingSummary.tsx

```diff
@@ -0,0 +1,117 @@
+import {
+  Badge,
+  Block,
+  Bold,
+  Flex,
+  List,
+  ListItem,
+  Metric,
+  ProgressBar,
+  Text,
+  Title
+} from '@tremor/react';
+import { AssessmentOverview, GradingStatuses } from 'src/commons/assessment/AssessmentTypes';
+import { GradingOverview } from 'src/features/grading/GradingTypes';
+
+import { AssessmentTypeBadge } from './GradingBadges';
+
+type GradingSummaryProps = {
+  group: string | null;
+  submissions: GradingOverview[];
+  assessments: AssessmentOverview[];
+};
+
+type AssessmentSummary = {
+  id: number;
+  type: string;
+  title: string;
+};
+
+const GradingSummary: React.FC<GradingSummaryProps> = ({ group, submissions, assessments }) => {
+  submissions = submissions.filter(({ assessmentType }) => assessmentType !== 'Paths');
+  const groupSubmissions = submissions.filter(
+    ({ groupName }) => group === null || groupName === group
+  );
+  const ungraded = groupSubmissions.filter(
+    ({ gradingStatus }) => gradingStatus !== GradingStatuses.graded
+  );
+  const ungradedAssessments = [...new Set(ungraded.map(({ assessmentId }) => assessmentId))].reduce(
+    (acc: AssessmentSummary[], assessmentId) => {
+      const assessment = assessments.find(assessment => assessment.id === assessmentId);
+      if (!assessment || assessment.type === 'Path') return acc;
```

Comment:
the check for `assessment.type === 'Path'` is no longer needed here since L31 already handles it?

Reply:
Yes, good catch!